package.json

```diff
@@ -111,6 +111,7 @@
     "eslint-plugin-jest": "~26.5.3",
     "eslint-plugin-prettier": "~4.0.0",
     "eslint-plugin-react": "~7.29.4",
+    "hatchling": "^0.2.2",
```

Comment:
```suggestion
```

Reply:
@athornton you will have to apply this suggestion yourself as I don't have rights to apply them myself.

Reply:
Done!